docs/src/content/docs/tools.mdx

```diff
@@ -302,15 +284,16 @@ The `definition` object in the tool creation and update requests follows the [Ba
 - `description` (string): A clear, concise description of what the tool does.
 - `dynamicParameters` (array, optional): List of parameters that can be set by the AI model when using the tool. Each parameter is an object containining:
   - `name` (string): The name of the parameter.
-  - `location` (string): Where the parameter is used (e.g., "PARAMETER_LOCATION_QUERY", "PARAMETER_LOCATION_PATH").
+  - `location` (string): Where the parameter is used ("PARAMETER_LOCATION_UNSPECIFIED", "PARAMETER_LOCATION_QUERY", "PARAMETER_LOCATION_PATH", "PARAMETER_LOCATION_HEADER", "PARAMETER_LOCATION_BODY").
```

Comment:
```suggestion
  - `location` (string): Where the parameter is used ("PARAMETER_LOCATION_QUERY", "PARAMETER_LOCATION_PATH", "PARAMETER_LOCATION_HEADER", "PARAMETER_LOCATION_BODY").
```

(unspecified isn't actually valid. It's a placeholder for "unset")

Reply:
Ya, makes sense.  FWIW all our enums will have an UNSPECIFIED value that's equivalent to not setting the field (and which is thus illegal if the field is required).  It's just a quirk of how OpenAPI gets generated from proto files.